### PR TITLE
Orders user profile

### PR DESCRIPTION
**Pull request by GregJMitchell**
This Branch Does:
- Added views for user profile order index
- Add view for user profile order show page
- Add cancel order function

### CODE REVIEW

File: spec/features/profile_orders/show_spec.rb [hunk 1]
```diff
@@ -197,81 +197,80 @@
...
+
+        new_order = Order.last
+
+        click_on new_order.id.to_s
+
+        click_on 'Cancel Order'
+
+        expect(current_path).to eq('/profile/orders')
+
+        expect(new_order.item_orders.all? { |item_order| item_order.status == 'unfulfilled' }).to eq(true)
```
Comment (abcdefghijohn) [inline spec/features/profile_orders/show_spec.rb:265]: Learned something new here, didn't know we could do it this way.

**Review by abcdefghijohn — APPROVED** [review]

File: app/controllers/profile_orders_controller.rb [hunk 2]
```diff
@@ -1,9 +1,18 @@
...
 
-  def index
-
+  def cancel
+    order = Order.find(params[:id])
+    order.update(status: 'cancelled')
+    order.return_items
+    order.item_orders.update(status: "unfulfilled")
+    flash[:notice] = "Your order is now cancelled"
+    redirect_to profile_orders_path
```
Comment (jakeheft) [inline app/controllers/profile_orders_controller.rb:16]: Is there a more restful way we could do this? We can address this when we get to refactoring

**Review by GarrettCottrell — APPROVED** [review]

File: app/models/order.rb [hunk 3]
```diff
@@ -1,4 +1,6 @@
-class Order <ApplicationRecord
+# frozen_string_literal: true
```
Comment (jakeheft) [inline app/models/order.rb:1]: What does this do?

**Review by jakeheft — APPROVED** [review]
Great work

**Pull request merged**